.travis.yml

```diff
@@ -16,15 +16,18 @@ env:
   - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
   - DJANGO="django>=1.11,<1.12" DATABASE_URL='postgres://postgres@127.0.0.1/freeradius_test'
   - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
-  - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
+  - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
+  - DJANGO="django>=2.0,<2.1" DATABASE_URL='postgres://postgres@127.0.0.1/freeradius_test'
+  - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
+  - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
 
 before_install:
   - pip install --no-cache-dir -U -r requirements-test.txt
   - ./runflake8
   - ./runisort
 
 install:
-  - pip install $DJANGO
+  - if [[ $TRAVIS_PYTHON_VERSION == "3.6" ]]; then pip install "django>=1.11,<2.1"; fi
```

Comment:
add `-U` otherwise django 2.0 is not installed because django-environ installs django 1.11.
Are you checking every travis step by expanding the information on the builds to see what is being installed?

Also add an explicit line for python 2.7 in which you install `django>=1.11,<2.0`